packages/utils/src/lib/reports/generate-md-report.unit.test.ts

```diff
@@ -135,31 +135,39 @@ describe('auditDetailsIssues', () => {
   it('should include source file', () => {
     expect(
       auditDetailsIssues([
-        { source: { file: 'index.js' }, severity: 'error' } as Issue,
+        { source: { file: '/index.js' }, severity: 'error' } as Issue,
       ])?.toString(),
-    ).toMatch('`index.js`');
+    ).toMatch('`/index.js`');
   });
 
-  it('should include source position startLine', () => {
+  it('should include source file linked with position (VS Code)', () => {
+    vi.stubEnv('TERM_PROGRAM', 'vscode');
     expect(
-      auditDetailsIssues([
-        {
-          source: {
-            position: {
-              startLine: 4,
+      auditDetailsIssues(
+        [
+          {
+            source: {
+              file: '/src/index.js',
+              position: {
+                startLine: 4,
+                startColumn: 7,
+              },
             },
-          },
-          severity: 'info',
-        } as Issue,
-      ])?.toString(),
-    ).toMatch(/\|\s*4\s*\|/);
+            severity: 'warning',
+          } as Issue,
+        ],
+        { outputDir: '/.code-pushup' },
+      )?.toString(),
+    ).toMatch('[`/src/index.js`](../src/index.js#L4)');
+    vi.unstubAllEnvs();
```

Comment:
This won't be reached if the `expect` assertion fails. Use an `afterAll`/`afterEach` hook to make the cleanup happens regardless of test result.

The same problem is present in `packages/utils/src/lib/reports/formatting.unit.test.ts`, so it might be best to add this hook to [`testing/test-setup/src/lib/reset.mocks.ts`](https://github.com/code-pushup/cli/blob/main/testing/test-setup/src/lib/reset.mocks.ts), it seems reasonable all include this for all tests automatically.